packages/aws-cdk-lib/aws-ecs/test/container-definition.test.ts

```diff
@@ -1157,6 +1157,58 @@ describe('container definition', () => {
     });
   });
 
+  test('can add docker label to the container definition', () => {
```

Comment:
Can you please add a test for `dockerLabels: {}`? (should `Match.absent()` in cfn)